lib/commands/setViewportSize.js

```diff
@@ -37,16 +36,19 @@ module.exports = function setViewportSize(size) {
         throw new ErrorHandler.CommandError('number or type of arguments don\'t agree with setViewportSize command');
     }
 
-    return setSize(this);
+    var shouldIndent = typeof(type) === 'undefined' ? true : type;
```

Comment:
I would like to have `false/undefined` as default so the default action is `windowHandleSize`. If someone set type to `true` it changes the viewport size